doc/lua_api.txt

```diff
@@ -5288,6 +5288,18 @@ Call these functions only at load time!
 * `minetest.register_on_item_eat(function(hp_change, replace_with_item, itemstack, user, pointed_thing))`
     * Called when an item is eaten, by `minetest.item_eat`
     * Return `itemstack` to cancel the default item eat response (i.e.: hp increase).
+* `minetest.register_on_item_pickup(function(itemstack, picker, pointed_thing, time_from_last_punch,  ...))`
+    * Called by `minetest.item_pickup` before an item is picked up.
+    * Function is added to `minetest.registered_on_item_pickups`.
+    * Oldest functions are called first.
+    * Parameters:
+      * `itemstack`: The `ItemStack` to be picked up.
+      * `picker`: `ObjectRef` or `nil`
+      * `pointed_thing`: The dropped item (a `"__builtin:item"` luaentity) as
+        `type="object"` `pointed_thing`.
+      * `time_from_last_punch, ...`: Other parameters from `luaentity:on_punch`.
+    * Return and itemstack to cancel the default item pick-up response (i.e.: adding
```

Comment:
```suggestion
    * Return an ItemStack to cancel the default item pick-up response (i.e.: adding
```

Reply:
`itemstack` is correct here, because it can also be a string.